base/build.gradle

```diff
@@ -32,6 +32,6 @@ dependencies {
     implementation Dependencies.coroutinesCore
     implementation Dependencies.androidxAnnotations
 
-    testImplementation Dependencies.junit4
+    testImplementation "junit:junit:4.13.2"
```

Comment:
We might not `base` module in future, but that's something to evaluate later. Added this to fix for now.